src/aerovaldb/jsondb/jsonfiledb.py

```diff
@@ -287,6 +289,8 @@ async def _get(
             k: v
             if isinstance(v, _LiteralArg)
             else encode_str(v, encode_chars=AerovalJsonFileDB.FNAME_ENCODE_CHARS)
+            if not isinstance(v, EncodedStr)
+            else v
```

Comment:
I get blind reading a dict comprehension with double if-else-if-else? And the identical code is used in _put below. Make a clearly documented and easily readable method `_prepare_substitutions`.